.prettierignore

```diff
@@ -5,4 +5,6 @@ webview-ui/build/
 package-lock.json
 src/core/prompts/system.ts
 src/core/prompts/model_prompts/claude4.ts
-evals/
\ No newline at end of file
+evals/
+docs/
+out/
```

Comment:
I think it makes sense to exclude `out` from the formatter but i'll let you confirm!